src/components/viewarticle_components/ArticleCardFull.jsx

```diff
@@ -1,6 +1,30 @@
+import { useState } from "react";
 import { toDaysMonthsYears } from "../../utils/formatTimeStamp";
+import { patchArticleLikes } from "../../utils/apiRequest";
 
 const ArticleCardFull = ({ article }) => {
+  const [currentVotes, setCurrentVotes] = useState(article.votes);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  const likeOnClickHandler = (event) => {
+    let increment = 0
+    if (event.target.id === 'like') {
+      increment = 1
+    } else {
+      increment = -1
+    }
```

Comment:
Nice way of making this dynamic, alternatively you can pass a value to the onClickHandler and use that